typed-racket-lib/typed-racket/infer/infer-unit.rkt

```diff
@@ -381,7 +381,11 @@
 
      (define s-seq (seq ss (rest->end s-rest)))
      (define t-seq (seq ts (rest->end t-rest)))
-     (and (null? s-kws)
+     (and (or (null? s-kws)
+              ;; if all keywords are optional, then we can just
```

Comment:
Is the `(null? s-kws)` unnecessary? (i.e. the `for/and` returns true whenever `(null? s-kws)` is true).